db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2024_05_22_041528) do
+ActiveRecord::Schema[7.1].define(version: 2024_06_03_195202) do
```

Comment:
I wasn't sure if I should commit this, because I was getting conflicts, but good to know!